CONTRIBUTING.md

```diff
@@ -84,7 +93,7 @@ There are three PR templates. The [default template](./.github/PULL_REQUEST_TEMP
 
 ### Requesting Reviews
 
-In order to accommodate the review process, the author of the PR must complete the author checklist
+In order to accommodate the review process, the author of the PR must complete the author checklist (from the pull request template)
 to the best of their abilities before marking the PR as "Ready for Review". If you would like to
```

Comment:
at the beginning we have "TL;RD". Here we go more in details. Do you think we should remove the section at the top?

Reply:
seems redundant to me, consider linking from the TL;DR from this section. 

(I always thought that TL;DR was acronym for "too long didn't read"... teach me otherwise)

Reply:
OK, I moved and merged the PR content from "TL;DR" to here.